cliconfig/config.go

```diff
@@ -57,6 +57,7 @@ type AuthConfig struct {
 type ConfigFile struct {
 	AuthConfigs map[string]AuthConfig `json:"auths"`
 	HttpHeaders map[string]string     `json:"HttpHeaders,omitempty"`
+	PsFormat    string                `json:"psFormat,omitempty"`
```

Comment:
I'm not sure we should have this here since this info, I think, is just specific for one command and should probably live in the ps.go file. If we ever get to the point where each command is an independent exe then the common/config-info shouldn't have any cmd specific info as that would break composibility.
Is there any reason this *must* be in here instead of ps.go?